Sources/LeafKit/LeafRenderer.swift

```diff
@@ -17,10 +17,20 @@ public protocol LeafCache {
         _ document: ResolvedDocument,
         on loop: EventLoop
     ) -> EventLoopFuture<ResolvedDocument>
+	
     func load(
         documentName: String,
         on loop: EventLoop
     ) -> EventLoopFuture<ResolvedDocument?>
+	
+    /// - return nil if cache entry didn't exist in the first place, true if purged
+    /// - will never return false in this design but should be capable of it
+    ///   in the event a cache implements dependency tracking between templates
+	@discardableResult func purge(
+        _ name: String,
```

Comment:
This should be `documentName`, too.

Reply:
Changed to documentName & no longer discardable